network/src/p2p/handler.rs

```diff
@@ -120,8 +121,11 @@ pub struct Handler {
     max_peers: usize,
     peer_db: Box<dyn (ManagingPeerdb)>,
     rng: Mutex<OsRng>,
+
+    event_sender: NotifySender,
```

Comment:
The names`NotifySender` and `event_sender` are too general.
It is hrad to know what they are using the name.
How about `informer_sender: InformerSender` or `informer_event_sender: InformerEventSender`?